Library/TelemetryClient.ts

```diff
@@ -217,6 +218,15 @@ class TelemetryClient {
             }
         }
 
+        // Sanitize tags and properties after running telemetry processors
+        if (accepted) {
+            envelope.tags = Util.validateStringMap(envelope.data.baseData.properties) as Tags & Tags[];
```

Comment:
You need to sanitize envelope.tags not baseData properties

Reply:
Good catch! I'll fix these.